pyro/contrib/epidemiology/compartmental.py

```diff
@@ -734,13 +734,11 @@ def _heuristic(self, haar, **options):
         assert isinstance(init_values, dict)
         assert "auxiliary" in init_values, \
             ".heuristic() did not define auxiliary value"
-        if haar:
-            haar.user_to_aux(init_values)
         logger.info("Heuristic init: {}".format(", ".join(
             "{}={:0.3g}".format(k, v.item())
             for k, v in sorted(init_values.items())
             if v.numel() == 1)))
-        return init_to_value(values=init_values)
+        return init_to_value(values=init_values, fallback=None)
```

Comment:
I've added fallback logic to a few init strategies (hoping to port these to NumPyro and combine with https://github.com/pyro-ppl/numpyro/pull/1058), and set the fallback to `None` here, which would error if the new initialization logic were to fail.